sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/model/ConsistencyPolicy.java

```diff
@@ -105,21 +107,21 @@ public ConsistencyPolicy setMaxStalenessPrefix(int maxStalenessPrefix) {
      *
      * @return the max staleness prefix.
      */
-    public int getMaxStalenessIntervalInSeconds() {
+    public Duration getMaxStalenessIntervalInSeconds() {
```

Comment:
hmm, do we want "InSeconds()" as suffix in the method name?

the time resolution for this method is seconds. If we are using Duration here we should update the documentation mentioning that the resolution is in seconds.

Reply:
We don't want inSeconds() as the suffix in the method name. I will remove it.